### ISSUE TITLE
Use the labelId for the name of the JSON

### ISSUE THREAD

**Issue by bustawin** [eReuse/device-inventory#56]
It is more useful to have the model name plus with the id of the label for the name of the JSON, making it easier to manage when creating labels, rather than the serial number (which could be a fallback if the label does not exist).

`fileName = modelName + '-' + labelId`

**Comment by ivilata** [comment]
Commit bb9e7ac7 uses the label if given, otherwise the previous nomenclature.

I'm not sure about including the model name, it may be too verbose. In any case I'd include the model after the label/serial so that the differentiating part comes first. What do you think?

**Comment by ivilata** [comment]
Assigning to @bustawin for feedback.

**Comment by bustawin** [comment]
@ivilata Organizations usually perform batches of computers of the same model, so if they have the model first and they order the files by name... you can see the point

**Comment by ivilata** [comment]
Yeah, it makes sense. Commit 989ae3b5 uses ``manufacturer,model,label-or-serial.json`` instead.
